### ISSUE TITLE
Hubspot connector: cannot sync custom objects

### ISSUE THREAD

**Issue by jtruty** [airbytehq/airbyte#26634]
### Connector Name

source-hubspot

### Connector Version

0.8.0

### What step the error happened?

During the sync

### Revelant information

With the hubspot connector v0.8.0 I can select a custom object I created, but I cannot successfully sync it via airbyte. The relevant error is related to getting the custom object properties:
```
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://api.hubapi.com/properties/v2/so_many_objects/properties
```
NOTE: so_many_objects is my custom object plural name
I believe this endpoint is not valid for custom objects. I think we need to use an updated v3 api to get at custom object properties: 
```
GET
/crm/v3/schemas/{objectType}
```
here are relevant docs: https://developers.hubspot.com/docs/api/crm/crm-custom-objects

### Relevant log output

```shell
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://api.hubapi.com/properties/v2/so_many_objects/properties
,retryable=<null>,timestamp=1685056569708,additionalProperties={}], io.airbyte.config.FailureReason@4097b9c1[failureOrigin=source,failureType=<null>,internalMessage=Source didn't exit properly - check the logs!,externalMessage=Something went wrong within the source connector,metadata=io.airbyte.config.Metadata@798d1bb5[additionalProperties={attemptNumber=2, jobId=121, connector_command=read}],stacktrace=io.airbyte.workers.internal.exception.SourceException: Source didn't exit properly - check the logs!
	at io.airbyte.workers.general.DefaultReplicationWorker.lambda$readFromSrcAndWriteToDstRunnable$8(DefaultReplicationWorker.java:442)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1589)
Caused by: io.airbyte.workers.exception.WorkerException: Source process exit with code 1. This warning is normal if the job was cancelled.
	at io.airbyte.workers.internal.DefaultAirbyteSource.close(DefaultAirbyteSource.java:150)
	at io.airbyte.workers.general.DefaultReplicationWorker.lambda$readFromSrcAndWriteToDstRunnable$8(DefaultReplicationWorker.java:440)
	... 4 more
```

### Contribute

- [ ] Yes, I want to contribute

**Comment by pquadri** [comment]
same issue here

**Comment by arsenlosenko** [comment]
@jtruty this issue was fixed in version 1.0.1 with this PR:
https://github.com/airbytehq/airbyte/pull/27658
Can you please confirm if you are still experiencing this issue? Until then I will close this ticket, you can re-open it if this problem is still occurring for you.
